recipes/default.rb

```diff
@@ -57,7 +57,13 @@
   ports %w(9323)
 end
 
-docker_installation_package 'default'
+docker_service 'default' do
+  node['osl-docker']['service'].each do |key, value|
+    send(key.to_sym, value)
+  end
+  install_method 'auto'
+  action :create
+end
```

Comment:
So I took another look at this and I think this might be a better way around it:
- Merge both resource definitions into this one here
- Make sure the `node.default` attributes below are above this resource
- Create a logic block that checks to see if certs exist or not, if they aren't there, then just do `:create`. If they do exist, then you can do `[:create, :start]`
- Add a `notifies :restart, 'docker_service[default]'` to all ssl related resources (it's already done on a few)

Does this make sense?

Reply:
That makes sense, I'll give it a try. That sounds like it will work!